.travis.yml

```diff
@@ -34,12 +34,13 @@ env:
 
 
 install:
-  - pip install tox codecov
+  - pip install tox codecov coverage
 
 script:
   - tox -e $TOX_ENV
 
 after_success:
+  - coverage combine
```

Comment:
I don't know what the rule is for combining multiple `after_success` items, but it would seem more straightforward to me to do `coverage combine && codecov` to ensure that it only happens in the positive-path case.